main.py

```diff
@@ -1,44 +1,249 @@
+	client = speech.SpeechClient()
+	config = {
+	        "language_code": language_code,
+	        "sample_rate_hertz": sample_rate_hertz,
+	        "enable_word_time_offsets": True,
+	        "encoding": encoding,
+	    }
+
+	with io.open(audio_path, "rb") as f:
+	    content = f.read()
+	audio = {"content": content}
+
+	my_results = []
+
+	response = client.long_running_recognize(config, audio).result()
+	flag = 0
+	for result in response.results:
+		flag = flag+1
+		alternative = result.alternatives[0]
+		if flag==1:
+			data = alternative.transcript
+		else:
+			tran = alternative.transcript
+			data = data +tran
+		#print('=' * 20)
+		#print('transcript: ' + alternative.transcript)
+		#print('confidence: ' + str(alternative.confidence))
+		for w in alternative.words:
+		    my_results.append({"word": w.word, 
+		       "st": w.start_time.seconds + float(w.start_time.nanos) / 10**9,
+		        "et": w.end_time.seconds + float(w.end_time.nanos) / 10**9
+		      })
+	with open("asrPrediction.txt", "w") as f: 
+		f.write(data) 
+	return my_results,data;
+
+def load_reference_data(path):
+	text = open(path).read()
+	return text
+
+def to_translation_map(iterable):						#a function to make a set of junk (None) characters
+    return {key: None for key in iterable}
+
+#-----------------------------OLD WAY VIA DIFFLIB/IT WILL BE REPLACED--------------------------------------------------
+def text_to_text_alignment_and_score(text_ref,text_pred):
+
+	text_ref = text_ref.lower()    #convert texts to consist of lower case characters only
+	text_pred = text_pred.lower()
+	iterable = [".",","]
+	translation_map = str.maketrans(to_translation_map(iterable)) #convert the reference text in order not to contain , and . (junk characters)
+	text_ref = text_ref.translate(translation_map)
+
+	# Create sequences to be aligned.
+	a = Sequence(text_ref.split())
+	b = Sequence(text_pred.split())
+	#print('Sequence A:', a)
+	#print('Sequence B:', b)
+	
+	# Create a vocabulary and encode the sequences.
+	v = Vocabulary()
+	aEncoded = v.encodeSequence(a)
+	bEncoded = v.encodeSequence(b)
+	#print('Encoded A:', aEncoded)
+	#print('Encoded B:', bEncoded)
+
+	# Create a scoring and align the sequences using global aligner.
+	scoring = SimpleScoring(1, 0)
+	aligner = GlobalSequenceAligner(scoring, 0)
+	f,score, encodeds = aligner.align(aEncoded, bEncoded,text_ref.split(),text_pred.split(), backtrace=True)
+	print(f)
+	#print(score)
+	print(encodeds)
+
+	# Iterate over optimal alignments and print them.
+	for encoded in encodeds:
+	    alignment = v.decodeSequenceAlignment(encoded)
+	    #print ('Alignment:','\n',alignment)
+	    recall_score=alignment.score*100/len(text_ref.split())
+	    precision_score=alignment.score*100/len(text_pred)
+	    #print ('Score:',Score ,'%')
+	return alignment,recall_score,precision_score
+	    
+	#return matches #first element is always from reference text and second from asr
+
+
+
+
+def silence_removal(audio_path):
```

Comment:
comment this function (since it is not used anywhere). this will be changed with coder from librasa and pyaudioanalysis in another task